src/components/Feed/HeroBanner.js

```diff
@@ -17,30 +26,31 @@ function HeroBanner({ onRequestNewProposal }) {
   const { account } = useWallet()
   const { layoutName } = useLayout()
 
-  const banner = BANNERS[layoutName]
-  const compactMode = layoutName === 'small' || layoutName === 'medium'
+  const mobileMode = layoutName === 'small'
+  const tabletMode = layoutName === 'medium'
+  const compactMode = mobileMode || tabletMode
+  const banner =
+    !account && tabletMode ? BANNERS.medium_full : BANNERS[layoutName]
 
   return (
     <div
       css={`
         height: fit-content;
-
-        ${!compactMode &&
-          `
-          top: ${3 * GU}px;
-          position: sticky;
-          margin-top: ${3 * GU}px;
-        `}
+        ${mobileMode && account && `margin-top: ${2 * GU}px;`}
       `}
     >
       <div
         css={`
           background: url(${banner.image}) no-repeat;
           background-size: cover;
-          width: ${compactMode ? '100%' : '327px'};
-          height: 0;
+
           padding-top: ${banner.aspectRatio};
           position: relative;
+          ${tabletMode &&
+            `
+          min-height: ${27 * GU}px;
+          ${!compactMode && `border-radius: ${BIG_RADIUS}px;`}      
```

Comment:
Shouldn't we set border radius if `account` ? 
Besides if `tabletMode` is true, `!compactMode` will always return false.

```suggestion
          ${account && `border-radius: ${BIG_RADIUS}px;`}      
```